apps/worker/main.py

```
@@ -0,0 +1,152 @@
+                "product_id": it.product_id,
+                "sku": it.sku,
+                "name": it.name,
+                "price": float(it.price),
+                "quantity": int(it.quantity),
+            } for it in oc.items]
+            return {
+                "order_id": oc.order_id or evt.order_id,
+                "customer": {"user_id": oc.customer.user_id, "email": oc.customer.email},
+                "items": items,
+            }
+        raise ValueError("not ORDER_CREATED")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)